pkg/operator/k8sutil/configmap.go

```diff
@@ -36,3 +37,13 @@ func DeleteConfigMap(clientset kubernetes.Interface, cmName, namespace string, o
 	defaultWaitOptions := &WaitOptions{RetryCount: 20, RetryInterval: 2 * time.Second}
 	return DeleteResource(delete, verify, resource, opts, defaultWaitOptions)
 }
+
+// GetConfigMap returns a configMap or error
+func GetConfigMap(clientset kubernetes.Interface, cmName, namespace string) (*corev1.ConfigMap, error) {
```

Comment:
To make this helper more helpful, how about this? It will simplify the code everywhere you need to get an operator setting.
```
func GetOperatorSetting(clientset kubernetes.Interface, namespace, settingName, defaultValue string) (string, error) {
  Get the settings configmap 
  If the configmap doesn't exist, look for the setting in env vars
  if the setting isn't found in env vars, return defaultValue
}
```